ibm_catalog.json

```diff
@@ -0,0 +1,146 @@
+{
+    "products": [
+        {
+            "features": [
+                {
+                    "description": "Supports multifactor authentication (MFA) for all users or federated users. Includes email-based, TOTP, and U2F MFA.",
+                    "title": "Multifactor authentication"
+                },
+                {
+                    "description": "Restrict API key creation (on-off)\nRestrict service ID creation (on-off)",
+                    "title": "Restrict API Key and Service IDs"
+                },
+                {
+                    "description": "Session activity timeout (seconds)\nSession inactivity timeout (seconds)",
+                    "title": "Session timeouts"
+                },
+                {
+                    "description": "Access token expiration (seconds)\nRefresh token expiration (seconds)",
+                    "title": "Token Expiration"
+                },
+                {
+                    "description": "off/Any Allowed - on/Allow only specified IP subnets or IP addresses. \n\nTwo control modes are supported Monitor \u0026 Restrict",
+                    "title": "Restrict IP address access"
+                },
+                {
+                    "description": "Global shell settings (on-off)\nPublic access group (on-off)",
+                    "title": "Global shell settings and public access group"
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "fscloud",
+                    "name": "fscloud",
+                    "architecture": {
+                        "descriptions": "",
+                        "features": [
+                            {
+                                "description": "Supports multifactor authentication (MFA) for all users or federated users. Includes email-based, TOTP, and U2F MFA.",
```

Comment:
If all flavors support these, they shouldn't be here. If only this flavor supports a feature, it should be here and not at the product level.